l10n_it_fatturapa_in/wizard/link_to_existing_invoice.py

```diff
@@ -10,10 +11,26 @@ class WizardLinkToInvoice(models.TransientModel):
     invoice_id = fields.Many2one(
         'account.invoice', string="Bill", required=True)
 
+    def get_invoice_obj(self, fatturapa_attachment):
+        xml_string = fatturapa_attachment.get_xml_string()
+        return fatturapa_v_1_2.CreateFromDocument(xml_string)
+
     @api.multi
     def link(self):
         self.ensure_one()
         active_ids = self.env.context.get('active_ids')
         if len(active_ids) != 1:
             raise UserError(_("You can select only one XML file to link."))
         self.invoice_id.fatturapa_attachment_in_id = active_ids[0]
+        # extract pdf if attached
+        fatturapa_attachment_obj = self.env['fatturapa.attachment.in']
```

Comment:
Avrei preferito `fattura_attachment_in_model` o semplicemente `fatturapa_attachment_in` in base a https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id33

Reply:
@tafaRU concordo. In questo caso, visto che le modifiche per le altre versioni sono già state rilasciate, lo terrei così, per ridurre il rischio di conflitti nel portare eventuali patches

Reply:
va bene